lib/utils/http-utils.js

```diff
@@ -4,11 +4,18 @@
 
 var networkClientFactory = require('../NetworkClientFactory');
 var configuration = require('../../common/Configuration');
-var HTTPUtils = function(logger){
+var qfs = require("q-io/fs");
+var Q = require("q");
+
+
+
+var HttpDownloadNotChanged="HttpDownloadNotChanged";
```

Comment:
Maybe its better to wrap HTTPUtils module (like log decorator)  with your extension, rather than hack it.